JortPob/Model/NIFToFLVER.cs

```diff
@@ -0,0 +1,340 @@
+                            flverVertex.UVs.Add(new Vector3(1f, 1f, 0f));
+                        }
+
+                        flverMesh.Vertices.Add(flverVertex);
+                        flverFaces.Indices.Add(flverMesh.Vertices.Count - 1);
+                    }
+
+                }
+
+                flver.Meshes.Add(flverMesh);
+            }
+
+            //for (int emmitterIndex = 0; emmitterIndex < nif.Emitters.Count; emmitterIndex++)
+            //{
+            //    var emmiter = nif.Emitters[emmitterIndex];
+
+                
+            //}
+
+            /* Add Dummy Polys */
+            short nextRef = 500; // idk why we start at 500, i'm copying old code from DS3 portjob here
+            List<Tuple<string, Vector3>> nodes = [
+                new("root", Vector3.Zero), // always add a dummy at root for potential use by fxr later
+            ];
+            foreach (Tuple<string, Vector3> tuple in nodes)
+            {
+                string name = tuple.Item1;
+                Vector3 position = tuple.Item2;
+
+                short refid = modelInfo.dummies.ContainsKey(name) ? modelInfo.dummies[name] : nextRef++;
+
+                // correct position using same math as we use for vertices above
+                position = position * Const.GLOBAL_SCALE;
+                position.X *= -1f;
+                Matrix4x4 rotateY180Matrix = Matrix4x4.CreateRotationY((float)Math.PI);
+                position = Vector3.Transform(position, rotateY180Matrix);
+
+                FLVER.Dummy dmy = new();
+                dmy.Position = position;
+                dmy.Forward = new(0, 0, 1);
+                dmy.Upward = new(0, 1, 0);
+                dmy.Color = System.Drawing.Color.White;
+                dmy.ReferenceID = refid;
+                dmy.ParentBoneIndex = 0;
+                dmy.AttachBoneIndex = -1;
+                dmy.UseUpwardVector = true;
+                flver.Dummies.Add(dmy);
+                if (!modelInfo.dummies.ContainsKey(name)) { modelInfo.dummies.Add(name, refid); }
+            }
+
+            /* Calculate bounding boxes */
+            BoundingBoxSolver.FLVER(flver);
+
+            /* Optimize flver */
+            flver = FLVERUtil.Optimize(flver);
+
+            /* Calculate model size */
+            float size = Vector3.Distance(rootNode.BoundingBoxMin, rootNode.BoundingBoxMax);
+            modelInfo.size = size;
+
+            /* Write flver */
+            flver.Write(outputPath);
+
+            /* Load overrides list for collision */
+            JsonNode json = JsonNode.Parse(File.ReadAllText(Utility.ResourcePath(@"overrides\static_collision.json")));
+            bool CheckOverride(string name)
+            {
+                foreach (JsonNode node in json.AsArray())
+                {
+                    if (node.ToString().ToLower() == name.ToLower()) { return true; }
+                }
+                return false;
+            }
```

Comment:
this is potentially very expensive. we can speed this up cheaply by using a `HashSet`:
```c#
JsonNode json = JsonNode.Parse(File.ReadAllText(Utility.ResourcePath(@"overrides\static_collision.json")));
HashSet<string> nodeNames = json.Select(node => node.ToString().ToLower()).ToHashSet();

// ... then, to check it:
            if ((nif.CollisionMeshes.Count > 0 || forceCollision) && !nodeNames.Contains(modelInfo.name.ToLower()))
            {
```